### PR TITLE
feat: Create vc store

### LINKED ISSUE #1049: Create vc store

### PR DESCRIPTION
**Pull request by fqutishat**
closes #1049

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: pkg/store/verifiable/store.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	store, err := ctx.StorageProvider().OpenStore(nameSpace)
+	if err != nil {
+		return nil, fmt.Errorf("failed to open vc store: %w", err)
+	}
+
+	return &Store{store: store}, nil
+}
+
+// SaveVC save verifiable credential
+func (s *Store) SaveVC(vc *verifiable.Credential) error {
```
Comment (troyronda) [inline pkg/store/verifiable/store.go]: I assume we will need to consider metadata and queries as well.
Is this a followup?

File: pkg/store/verifiable/store_test.go [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+  "refreshService": {
+    "id": "https://example.edu/refresh/3732",
+    "type": "ManualRefreshService2018"
+  }
+}
+`
+
+func TestNew(t *testing.T) {
+	t.Run("test new store", func(t *testing.T) {
+		_, err := New(&mockprovider.Provider{
```
Comment (sandrask) [inline pkg/store/verifiable/store_test.go]: verify store is not nil

File: pkg/store/verifiable/store_test.go [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+			StorageProviderValue: mockstore.NewMockStoreProvider()})
+		require.NoError(t, err)
+	})
+
+	t.Run("test error from open store", func(t *testing.T) {
+		_, err := New(&mockprovider.Provider{
```
Comment (sandrask) [inline pkg/store/verifiable/store_test.go]: verify store is nil in case of error

File: pkg/store/verifiable/store.go (hunk 1 above)
Comment (fqutishat) [reply pkg/store/verifiable/store.go]: queries will in followup PR

File: pkg/store/verifiable/store_test.go (hunk 2 above)
Comment (fqutishat) [reply pkg/store/verifiable/store_test.go]: done

File: pkg/store/verifiable/store_test.go (hunk 3 above)
Comment (fqutishat) [reply pkg/store/verifiable/store_test.go]: done

**Pull request merged**